TestProjects/VisualEffectGraph_HDRP/ProjectSettings/EditorBuildSettings.asset

```diff
@@ -241,10 +239,10 @@ EditorBuildSettings:
   - enabled: 1
     path: Assets/AllTests/VFXTests/GraphicsTests/31_CameraBuffer.unity
     guid: 022b924fd58e94e41bcc5a24218370fe
-  - enabled: 0
-    path: Assets/AllTests/VFXTests/GraphicsTests/32_ExcludeFromTAA.unity
-    guid: 8f9c50f62a67d6d429a71cb9ad4a3f57
```

Comment:
This test has been temporarily removed, you should resolve the editor settings change manually : There is a proess which is removing unchecked test in editorSettings